1 - ----------T--:--:--.--- Python Client Error Trace
1 - ----------T--:--:--.--- Python Client Main - Caught an ACSException...don't worry because this SHOULD happen.
1 - ----------T--:--:--.--- Python Client Main - Trying to invoke bad method
1 - ErrorTrace (TimeStamp=xxxx,
1 -             File=xxx
1 -             Line=xxx
1 -             Routine=<module>,
1 -             Host=xxxx,
1 -             Process=xxx,
1 -             Thread=MainThread,
1 -             Type=10,
1 -             Code=3,
1 -             ShortDescrip=Unknown Error,
1 -             Severity=Error,
1 -             Data: )
1 - 
1 -     ErrorTrace (TimeStamp=xxxx,
1 -                 File=xxx
1 -                 Line=xxx
1 -                 Routine=HelloWorld::badMethod,
1 -                 Host=xxxx,
1 -                 Process=xxxTestContainer,
1 -                 Thread=ORBTask,
1 -                 Type=10,
1 -                 Code=3,
1 -                 ShortDescrip=Unknown Error,
1 -                 Severity=Error,
1 -                 Data: )
1 - 
1 - ErrorTrace (TimeStamp=xxxx,
1 -             File=xxx
1 -             Line=xxx
1 -             Routine=<module>,
1 -             Host=xxxx,
1 -             Process=xxx,
1 -             Thread=MainThread,
1 -             Type=10,
1 -             Code=3,
1 -             ShortDescrip=Unknown Error,
1 -             Severity=Error,
1 -             Data: )
1 - 
1 -     ErrorTrace (TimeStamp=xxxx,
1 -                 File=xxx
1 -                 Line=xxx
1 -                 Routine=HelloWorld::badMethod,
1 -                 Host=xxxx,
1 -                 Process=xxxTestContainer,
1 -                 Thread=ORBTask,
1 -                 Type=10,
1 -                 Code=3,
1 -                 ShortDescrip=Unknown Error,
1 -                 Severity=Error,
1 -                 Data: )
1 - 
1 - ----------T--:--:--.--- Python Client disconnect - Shutdown called for client
1 - The end __oOo__
